.github/workflows/corpus.yml

```diff
@@ -26,7 +30,7 @@ jobs:
         run: go test -count=1 -v -tags corpus ./integration_tests/... | tail
 
       - name: Notify on Failure
-        if: failure()
+        if: failure() && github.event_name == 'schedule'
```

Comment:
We only want to be notified when this fails in the nightly run. Manual invocations shouldn't file an issue.